packages/sn-redux/src/Actions.ts

```diff
@@ -122,6 +122,7 @@ import {
 } from '@sensenet/client-core'
 import { GenericContent, User } from '@sensenet/default-content-types'
 import { PromiseMiddlewareAction } from '@sensenet/redux-promise-middleware'
+import { PathHelper } from '@sensenet/client-utils'
```

Comment:
Please add client-utils to package.json and to tsconfig.json as well.

Reply:
I've added it to the package.json and I noticed that it was already there in the tsconfig's references array.  Is it okay there, or it should be added to one of the other lists there (e.g. includes)?

Reply:
I can't see sn-client-utils inside the references. It should be there.